transports/tls/CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## 0.7.0
```

Comment:
I could be missing something, but looking at the tls changes, i dont believe we would need to do a minor bump. This should be able to be under  `0.6.1`, no?